vault/barrier_aes_gcm.go

```diff
@@ -708,6 +708,11 @@ func (b *AESGCMBarrier) Get(ctx context.Context, key string) (*Entry, error) {
 		return nil, nil
 	}
 
+	if pe.Value == nil {
```

Comment:
Or should the condition be `len(pe.Value) < 4`?

Reply:
I think we should check for both. Added another check.

Reply:
You don't need both. `len(pe.Value) < 4` will also match when nil.

Reply:
Yeah i think all we need is `len(pe.Value) < 4`

Reply:
Yeah, right. Fixed.